alica_engine/include/engine/AlicaContext.h

```diff
@@ -70,7 +70,10 @@ struct AlicaContextParams
      * @param roleSetName Name of the roleSet.
      * @param masterPlanName Name of the masterPlan
      * @param stepEngine Signify engine is trigger based. Defaults to false.
-     * @param agentID Identifier of the local Agent. By default a new identifier is created.
+     * @param agentID Identifier of the local Agent. If no identifier is given, an
+     * invalid/temporary identifier is created and therefore, the engine will try to read the local
```

Comment:
Please remove this part "an invalid/temporary identifier is created and therefore" from comments for both constructors

Reply:
done